people.json

```
@@ -5690,4 +5690,27 @@
          ]
       ]
    }
```

Looks like we're missing a comma here - that'll cause errors when the script tries to parse this file:
```suggestion
   },
```